doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -92,8 +92,9 @@ Developing with PMICs
 
 Security
 ========
+* Updated:
```

Comment:
I think there needs to be an empty line between the header and text.

Reply:
Leave out the Updated line.